src/pages/policy/output/Display.jsx

```diff
@@ -0,0 +1,333 @@
+      policy={policy}
+      hasShownPopulationImpactPopup={hasShownPopulationImpactPopup}
+      setHasShownPopulationImpactPopup={setHasShownPopulationImpactPopup}
+    >
+      {pane}
+    </LowLevelDisplay>
+  );
+}
+
+/**
+ *
+ * This component makes the component it wraps around screenshottable. It also
+ * adds buttons for performing actions such as downloading data and sharing
+ * results at the top, an informational carousel at the bottom, and an
+ * informational popup. This component should not be used directly under normal
+ * circumstances. It is already a part of the higher-level component
+ * DisplayImpact: the higher-level component should be used whenever possible.
+ *
+ * @param {object} props
+ * @param {function} props.downloadCsv callback for download csv button
+ * @param {function} props.downloadPng callback for download png button
+ * @param {object} props.policy the policy object
+ * @param {object} props.metadata the metadata object
+ * @param {boolean} props.hasShownPopulationImpactPopup indicator
+ * @param {function} props.setHasShownPopulationImpactPopup setter for indicator
+ * @returns a component that adds decorations around a child component that is
+ * the main object, e.g., a chart
+ */
+export function LowLevelDisplay(props) {
+  const {
+    children,
+    downloadCsv,
+    downloadPng,
+    metadata,
+    policy,
+    hasShownPopulationImpactPopup,
+    setHasShownPopulationImpactPopup,
+  } = props;
+  const mobile = useMobile();
+  const [preparingForScreenshot, setPreparingForScreenshot] = useState(false);
+  const [, takeScreenShot] = useScreenshot();
+  const imageRef = useRef(null);
+  useEffect(() => {
+    if (preparingForScreenshot) {
+      setTimeout(() => {
+        takeScreenShot(imageRef.current).then((img) => {
+          setPreparingForScreenshot(false);
+          // send a request to /image with the image
+          // The filename should be the current path (including query strings), but with /, &, ? etc. replaced with _
+          const filename =
+            (window.location.pathname + window.location.search)
+              .replaceAll("/", "_")
+              .replaceAll("&", "_")
+              .replaceAll("?", "_")
+              .replaceAll("=", "_")
+              .replaceAll(".", "_") + ".png";
+          fetch("/image", {
+            method: "POST",
+            body: JSON.stringify({
+              filename,
+              image: img,
+            }),
+            headers: {
+              "Content-Type": "application/json",
+            },
+          }).then((response) => {
+            if (response.ok) {
+              return response.json();
+            } else {
+              throw new Error("Something went wrong");
+            }
+          });
+        });
+      }, 1000);
+    }
+  }, [preparingForScreenshot, takeScreenShot]);
+
+  const urlParams = new URLSearchParams(window.location.search);
+  const selectedVersion = urlParams.get("version") || metadata.version;
```

Comment:
```suggestion
  const focus = urlParams.get("focus");
  const selectedVersion = urlParams.get("version") || metadata.version;
```

Reply:
Without including this, JS interprets the `focus` param in the `BottomCarousel` component as the literal JS function `focus`